src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/CodeDirectiveFormattingTest.cs

```diff
@@ -37,6 +37,44 @@ public interface Bar
                     """);
     }
 
+    [Fact]
+    public async Task FormatsCodeBlockDirectiveWithCRLFLineEnding()
```

Comment:
These tests are great, and I was going to ask you to create a version of the `Formats_CodeBlockDirectiveWithMarkup_NonBraced` test as well (because it adds a newline with trailing whitespace, which these don't cover), but then I thought about it more, and I think it would be more interesting if `FormattingTestBase` was modified to replace `\r\n` with `\n`, and we run every test with both line endings.

ie, duplicate these two lines again, but with `input` and `expected` having had their line endings replaced with LF
https://github.com/dotnet/razor/blob/main/src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/FormattingTestBase.cs#L54

Reply:
This makes sense! That would be a better way to widen the test coverage